Automattic/wp-calypso
# Fix the styles of Jetpack Cloud Settings section

#### Changes proposed in this Pull Request

Fix a few styles of the Jetpack Cloud Settings section:

**Asana Card:** 1142395350490785-as-1173118334902107

- Set the title font with the correct size
- Added the expected icons for connected and disconnected server status. Align it to the center.
- Fixed a line jump on disconnected status
- Added loading block with pulse effect

I copied completely the RewindCredentialsForm component from Calypso to be able to change styles and the icon position. Also, this allows us to have control of the design.

**Before**

![image](https://user-images.githubusercontent.com/9832440/80820600-1d0b6a80-8bcf-11ea-833e-ef4b76f9bfa9.png)

![image](https://user-images.githubusercontent.com/9832440/80820633-2eed0d80-8bcf-11ea-821d-63f1b2653a67.png)

![image](https://user-images.githubusercontent.com/9832440/80820658-38767580-8bcf-11ea-996d-3ba46a3b20a4.png)


**Now**

Loading block:
![image](https://user-images.githubusercontent.com/9832440/80820219-627b6800-8bce-11ea-82c2-51cb1e83430e.png)

The expected icons, centered:
![image](https://user-images.githubusercontent.com/9832440/80819821-a457de80-8bcd-11ea-984a-0cd403f73cf4.png)
![image](https://user-images.githubusercontent.com/9832440/80819769-8b4f2d80-8bcd-11ea-98d5-78897f68d583.png)


#### Testing instructions

- Apply these changes
- Check if the design match with the expected
